src/test/resources/projects/basic/archetype.properties

```diff
@@ -20,4 +20,5 @@ sdkFormsVersion=latest
 datalayer=y
 amp=y
 enableDynamicMedia=y
+archetypeVersion=28-SNAPSHOT
```

Comment:
Will this need to be updated manually after each release? Can we make it optional? It will always be present from the invocation anyway